airflow/providers/mysql/operators/mysql.py

```diff
@@ -46,6 +46,7 @@ class MySqlOperator(BaseOperator):
     """
 
     template_fields = ('sql',)
+    template_fields_renderers = {'sql': 'sql'}
```

Comment:
Nice, but we need a unit test here.

Reply:
Should we add `json` renderer for `parameters`?

Reply:
Interesting point, I don't think we have test for `template_fields_renderes` yet. What we probably should check is that renderers are proper values:
https://github.com/apache/airflow/blob/01fb7775ddcf8fdaa9afbfd60d66cc616a8f5226/airflow/www/utils.py#L351-L370

Reply:
Right. That works for me. Can we see the screenshot ?

Reply:
Please, check the screenshots:
![image](https://user-images.githubusercontent.com/33004847/125123912-fd8b9e00-e0ff-11eb-9cca-ff9dc3ebcd2b.png)
Test DAG:
![image](https://user-images.githubusercontent.com/33004847/125124023-21e77a80-e100-11eb-8370-d7f3d133864f.png)
Rendered Template:
![image](https://user-images.githubusercontent.com/33004847/125124082-33c91d80-e100-11eb-9ee0-b6a41a847ca9.png)